### PR TITLE
Various dotnet changes

### PR DESCRIPTION
**Pull request by khellang**
By request from @richlander :wink:
- Renamed ".NET Core" to ".NET Core Framework"
- Added .NET Core CLR
- Fixed link for .NET Core Framework

I also removed some trailing whitespace. The diff's probably best viewed using `?w=1` :smile:

### CODE REVIEW

**Comment by shiftkey** [comment]
The whitespace changes will likely be nuked once I go back to #160, but the rest is golden.

Thanks!

**Comment by richlander** [comment]
SWEEEEEEEEEEEEEET!

Thanks @khellang! Very much appreciated. Thanks for taking it @shiftkey.

**Comment by jrusbatch** [comment]
:clap:

**Comment by khellang** [comment]
Was about to add https://github.com/Microsoft/msbuild as well, but saw it didn't have an up-for-grabs label yet. It does have the default [`help wanted`](https://github.com/Microsoft/msbuild/labels/help%20wanted), though. But I guess those default labels will be switched out soon.

File: _data/projects.yml [hunk 1]
```diff
@@ -731,19 +731,27 @@
...
   site: https://github.com/dotnet/corefx/
-  tags: [ .NET ] 
+  tags: [ .NET ]
   upforgrabs:
     name: up-for-grabs
-    link: https://github.com/dotnet/corefx/labels/up-for-grabs
+    link: https://github.com/dotnet/corefx/labels/up%20for%20grabs
+
+- name: .NET Core CLR
+  desc: Contains the .NET Core foundational libraries, called CoreFX. It includes classes for collections, file systems, console, XML, async and many others.
```
Comment (akoeplinger) [inline _data/projects.yml:744]: @khellang that description sounds wrong for coreclr :)

Comment (khellang) [inline _data/projects.yml:744]: Ooops. Copy-pasta FAIL :sob: #178

**Comment by richlander** [comment]
Good idea, but up to @AndyGerlicher, who owns the repo.

**Comment by AndyGerlicher** [comment]
I added an up-for-grabs label on the msbuild repo. Thanks for setting this up!

**Pull request merged**